src/HttpServer.php

```diff
@@ -17,20 +17,26 @@ class HttpServer extends EventEmitter
 
 	private $socket;
 	private $callback;
+	private $errorResponse;
 
 	/**
 	 *
 	 * @param ServerInterface $socket - the server runs on this socket (ip address and port)
 	 * @param callable $callback - callback function which returns a RingCentral\Psr7\Response
 	 */
-	public function __construct(ServerInterface $socket, callable $callback)
+	public function __construct(ServerInterface $socket, callable $callback, Response $errorResponse = null)
```

Comment:
Functionally LGTM :+1: 

Though I wonder what the use case would be here? When would I set a custom response message here? And then, it's my understand one would want a custom response message depending on the incoming request (such as localizing the error message)?